index.js

```diff
@@ -74,7 +74,7 @@ class Provider {
   _parseArgs(uri) {
     // XXX: Reimplement querystring.parse to not escape
     const parsed = {}
-    const [producer, parsed] = uri.split('?')
+    const [ , parsed ] = uri.split('?')
```

Comment:
This wouldn't work because the `parsed` variable has already been defined in the line above, resulting in a:  `SyntaxError: Identifier 'parsed' has already been declared`